numpy/f2py/auxfuncs.py

```diff
@@ -88,187 +126,221 @@ def get_kind(var):
     if not isscalar(var):
         return 0
-    if not var.get('typespec')=='real':
+    if not var.get('typespec') == 'real':
         return 0
-    return get_kind(var)=='8'
+    return get_kind(var) == '8'
+
 
 def islong_double(var):
     if not isscalar(var):
         return 0
-    if not var.get('typespec')=='real':
+    if not var.get('typespec') == 'real':
         return 0
-    return get_kind(var)=='16'
+    return get_kind(var) == '16'
+
 
 def islong_complex(var):
     if not iscomplex(var):
         return 0
-    return get_kind(var)=='32'
+    return get_kind(var) == '32'
+
 
 def iscomplexarray(var):
     return isarray(var) and var.get('typespec') in ['complex', 'double complex']
 
+
 def isint1array(var):
-    return isarray(var) and var.get('typespec')=='integer' \
-    and get_kind(var)=='1'
+    return isarray(var) and var.get('typespec') == 'integer' \
+        and get_kind(var) == '1'
+
 
 def isunsigned_chararray(var):
     return isarray(var) and var.get('typespec') in ['integer', 'logical']\
-           and get_kind(var)=='-1'
+        and get_kind(var) == '-1'
+
 
 def isunsigned_shortarray(var):
     return isarray(var) and var.get('typespec') in ['integer', 'logical']\
-           and get_kind(var)=='-2'
+        and get_kind(var) == '-2'
+
 
 def isunsignedarray(var):
     return isarray(var) and var.get('typespec') in ['integer', 'logical']\
-           and get_kind(var)=='-4'
+        and get_kind(var) == '-4'
+
 
 def isunsigned_long_longarray(var):
     return isarray(var) and var.get('typespec') in ['integer', 'logical']\
-           and get_kind(var)=='-8'
+        and get_kind(var) == '-8'
+
 
 def issigned_chararray(var):
     return isarray(var) and var.get('typespec') in ['integer', 'logical']\
-           and get_kind(var)=='1'
+        and get_kind(var) == '1'
+
 
 def issigned_shortarray(var):
     return isarray(var) and var.get('typespec') in ['integer', 'logical']\
-           and get_kind(var)=='2'
+        and get_kind(var) == '2'
+
 
 def issigned_array(var):
     return isarray(var) and var.get('typespec') in ['integer', 'logical']\
-           and get_kind(var)=='4'
+        and get_kind(var) == '4'
+
 
 def issigned_long_longarray(var):
     return isarray(var) and var.get('typespec') in ['integer', 'logical']\
-           and get_kind(var)=='8'
+        and get_kind(var) == '8'
+
 
 def isallocatable(var):
     return 'attrspec' in var and 'allocatable' in var['attrspec']
 
+
 def ismutable(var):
-    return not (not 'dimension' in var or isstring(var))
+    return not ('dimension' not in var or isstring(var))
+
 
 def ismoduleroutine(rout):
     return 'modulename' in rout
 
+
 def ismodule(rout):
-    return ('block' in rout and 'module'==rout['block'])
+    return ('block' in rout and 'module' == rout['block'])
+
 
 def isfunction(rout):
-    return ('block' in rout and 'function'==rout['block'])
+    return ('block' in rout and 'function' == rout['block'])
+
+# def isfunction_wrap(rout):
```

Comment:
Yes, but with this code I'm never sure if commented out code might be informative as to intent.